src/BenchmarkDotNet/Analysers/OutliersAnalyser.cs

```diff
@@ -77,17 +80,19 @@ string Format(int n, string verb)
         }
 
         [CanBeNull]
-        private static string GetRangeMessage([NotNull] double[] values)
+        private static string GetRangeMessage([NotNull] double[] values, CultureInfo cultureInfo)
         {
+            string Format(double value) => TimeValue.FromNanoseconds(value).ToString(cultureInfo, "N2");
```

Comment:
@adamsitnik I don't think that we should create an additional method in `TimeValue` (aka `TimeInterval`) because we already have `ToString`. Here I introduce a local function just to remove duplications of `(cultureInfo, "N2")`.